admin/templates/nodes/storage_status.html

```diff
@@ -0,0 +1,11 @@
+{% if not resource.is_public and resource.storage_limit_status is STORAGE_LIMITS.APPROACHING_PRIVATE.value %}
```

Comment:
I'm afraid this still isn't working for me. 

<img width="403" alt="Screen Shot 2020-09-21 at 2 21 17 PM" src="https://user-images.githubusercontent.com/6599111/93805724-17fa9680-fc16-11ea-8602-b416ee50d9ff.png">